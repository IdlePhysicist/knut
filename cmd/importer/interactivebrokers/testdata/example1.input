Statement,Header,Field Name,Field Value
Statement,Data,BrokerName,Interactive Brokers
Statement,Data,BrokerAddress,
Statement,Data,Title,Activity Statement
Statement,Data,Period,"January 1, 2020 - April 22, 2020"
Account Information,Data,Base Currency,EUR
Open Positions,Data,Summary,Stocks,USD,AAPL,10,1,100.00,100.00,100.00,100.00,100.00,100.00,
Forex Balances,Data,Forex,CHF,CHF,320.072033121,1,-320.072033121,1,320.072033121,0,
Trades,Data,Order,Stocks,USD,AAPL,"2020-04-12, 10:17:49",7,10.00,10.00,-1.00,-1.00,0,0,40.425,O
Trades,Data,Order,Forex,USD,CHF.USD,"2020-03-21, 11:17:34","-4,59",1.03371,,449.38889,-1.1,,,,3.446,
Deposits & Withdrawals,Data,CHF,2020-01-20,Electronic Fund Transfer,1000
Dividends,Data,USD,2020-02-13,AAPL(US0378331005) Cash Dividend USD 0.77 per Share (Ordinary Dividend),10.00
Withholding Tax,Data,USD,2020-02-13,AAPL(US0378331005) Cash Dividend USD 0.77 per Share - US Tax,-1.23,