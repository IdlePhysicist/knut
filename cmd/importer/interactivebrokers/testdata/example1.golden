2020-01-20 "Deposit 1000 CHF"
Expenses:TBD     Assets:IB              1000 CHF

2020-02-13 "AAPL(US0378331005) Cash Dividend USD 0.77 per Share (Ordinary Dividend)"
Income:Dividends Assets:IB                10 USD
Income:Dividends Income:Dividends          0 AAPL

2020-02-13 "AAPL(US0378331005) Cash Dividend USD 0.77 per Share - US Tax"
Assets:IB        Expenses:Tax           1.23 USD
Expenses:Tax     Expenses:Tax              0 AAPL

2020-03-21 "Sell -459 CHF @ 1.03371 USD"
Assets:IB        Equity:Equity           459 CHF
Equity:Equity    Assets:IB            449.39 USD
Assets:IB        Expenses:Fees           1.1 EUR

2020-04-12 "Buy 7 AAPL @ 10 USD"
Equity:Equity    Assets:IB                 7 AAPL
Assets:IB        Equity:Equity             1 USD
Assets:IB        Expenses:Fees             1 USD

2020-04-22 balance Assets:IB 10 AAPL
2020-04-22 balance Assets:IB 320.07 CHF
